php/pooldao.class.inc

```diff
@@ -101,17 +101,12 @@ class PoolDAO extends \LORIS\Data\ProvisionerInstance
                   FROM   biobank_pool bp
                   LEFT JOIN biobank_specimen_pool_rel bspr
                     USING (PoolID)";
-        if (!empty($conditions)) {                                                 
-            $query .= ' WHERE ';                                                   
-            $whereClause = '';                                                     
-            foreach($conditions as $key=>$condition) {                             
-                reset($conditions);                                                
-                if ($key !== key($conditions)) {                                   
-                    $whereClause .= ' '.$condition['option'].' ';                  
-                }                                                                  
-                $whereClause .= $condition['column'].'='.'"'.$condition['value'].'"';
-            }                                                                      
-            $query .= $whereClause;                                                
+        if (!empty($conditions)) {
+            $whereClause = [];
+            foreach ($conditions as $column => $value) {
+                $whereClause[] = $column.'='.'"'.$value.'"';
+            }
+            $query .= ' WHERE '.implode(' AND ', $whereClause);
```

Comment:
@HenriRabalais this is actually better than the modified version so the revert is better.

Reply:
@ridz1208 what do you mean by "better"